gulpfile.js

```diff
@@ -0,0 +1,11 @@
+var gulp = require('gulp');
+gulp.task('generate-service-worker', function(callback) {
+  var path = require('path');
+  var swPrecache = require('sw-precache');
+  var rootDir = 'scripts';
+
+  swPrecache.write(path.join(rootDir, 'sw.js'), {
+    staticFileGlobs: [rootDir + '/**/*.{js,html,css,png,jpg,gif}'],
```

Comment:
@ngokevin So just caching this directory: https://github.com/aframevr/aframe-site/tree/master/themes/aframe/source ?